operations/compute_rolling_upgrades/Upgrade_Compute_Nodes_with_CRUS.md

```diff
@@ -90,23 +91,24 @@ Upgrade a set of compute nodes with the Compute Rolling Upgrade Service \(CRUS\)
     workload_manager_type = "slurm"
     ```
 
-    If successful, note the upgrade\_id in the returned data.
+1. Note the `upgrade_id` in the returned data of the previous command.
 
     ```bash
-    ncn-w001# export UPGRADE_ID=e0131663-dbee-47c2-aa5c-13fe9b110242
+    ncn# UPGRADE_ID=e0131663-dbee-47c2-aa5c-13fe9b110242
     ```
 
-5.  Monitor the status of the upgrade session.
+1. Monitor the status of the upgrade session.
 
-    The progress of the session through the upgrade process is described in the messages field of the session. This is a list of messages, in chronological order, containing information about stage transitions, step transitions, and other conditions of interest encountered by the session as it progresses. It is cleared once the session completes.
+    The progress of the session through the upgrade process is described in the `messages` field of the session. This is a list of messages, in chronological order, containing
+    information about stage transitions, step transitions, and other conditions of interest encountered by the session as it progresses. It is cleared once the session completes.
 
     ```bash
-    ncn-w001# cray crus session describe $UPGRADE_ID
+    ncn# cray crus session describe $UPGRADE_ID
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815